### ISSUE TITLE
Cleanup vendor directory

### ISSUE THREAD

**Issue by alexsegura** [coopcycle/coopcycle-web#678]
The vendor directory takes approx. ~ 200MB, even when using `--prefer-dist`. 

We should use some plugin to cleanup unnecessary files. Some options: 
- [Drupal Vendor Cleanup](https://www.drupal.org/docs/develop/using-composer/using-drupals-vendor-cleanup-composer-plugin) ([repo](https://github.com/drupal/drupal/tree/8.8.x/composer/Plugin/VendorHardening))
- [Victor The Cleaner](https://github.com/dg/composer-cleaner)
- [Composer Cleanup Plugin](https://github.com/barryvdh/composer-cleanup-plugin)
- [MediaMonks Composer Vendor Cleaner](https://github.com/MediaMonks/composer-vendor-cleaner)
- [Awesome Composer](https://github.com/jakoch/awesome-composer)

**Comment by alexsegura** [comment]
Here is a list of the biggest packages (more than 1MB)

```
63892   vendor/symfony
20560   vendor/giggsey
12380   vendor/api-platform
11804   vendor/fzaninotto
11356   vendor/sylius
9712    vendor/imagine
9276    vendor/doctrine
6464    vendor/azuyalabs
5984    vendor/nelmio
4924    vendor/twig
4556    vendor/nesbot
4080    vendor/jms
3724    vendor/composer
3584    vendor/friendsofsymfony
2596    vendor/swiftmailer
2340    vendor/gedmo
1792    vendor/zendframework
1740    vendor/jsor
1652    vendor/predis
1648    vendor/hwi
1640    vendor/php-http
1540    vendor/knplabs
1444    vendor/willdurand
1444    vendor/jmose
1316    vendor/ocramius
1208    vendor/liip
1200    vendor/illuminate
1196    vendor/monolog
1176    vendor/predicthq
1156    vendor/mustache
1032    vendor/league
1028    vendor/box
```

**Comment by alexsegura** [comment]
List of packages with most files

```
$ du --inodes -s vendor/* | sort -rh | head -n 50

9268	./symfony/symfony
1620	./api-platform/core
1112	./giggsey/libphonenumber-for-php
1109	./nelmio/alice
881	./nesbot/carbon
671	./jms/serializer
578	./sylius/resource-bundle
542	./fzaninotto/faker
509	./doctrine/orm
414	./gedmo/doctrine-extensions
410	./swiftmailer/swiftmailer
408	./twig/twig
400	./friendsofsymfony/rest-bundle
390	./jsor/doctrine-postgis
363	./hwi/oauth-bundle
326	./predis/predis
301	./doctrine/dbal
289	./giggsey/locale
276	./friendsofsymfony/user-bundle
260	./liip/imagine-bundle
248	./sylius/product-bundle
248	./mustache/mustache
240	./hautelook/alice-bundle
229	./sylius/promotion-bundle
228	./azuyalabs/yasumi
214	./ocramius/proxy-manager
205	./lexik/jwt-authentication-bundle
204	./monolog/monolog
197	./box/spout
192	./oneup/uploader-bundle
187	./behat/transliterator
186	./sylius/attribute-bundle
176	./sylius/order-bundle
173	./sylius/payment-bundle
172	./willdurand/hateoas
171	./sensio/framework-extra-bundle
165	./beberlei/doctrineextensions
164	./martin-georgiev/postgresql-for-doctrine
158	./trikoder/oauth2-bundle
156	./knplabs/knp-components
155	./sylius/channel-bundle
155	./imagine/imagine
149	./sylius/currency-bundle
148	./stripe/stripe-php
145	./intervention/image
143	./sylius/taxation-bundle
140	./vich/uploader-bundle
140	./sylius/locale-bundle
137	./sylius/taxonomy-bundle
127	./illuminate/contracts
```

**Comment by alexsegura** [comment]
Fixed in 44e8b78efdf12827241f5bfe164761007f46f546
